kitsune/sumo/static/sumo/js/kbox.js

```diff
@@ -79,7 +79,7 @@
   '<div class="kbox-title"></div>' +
   '<div class="kbox-wrap"><div class="kbox-placeholder"/></div>' +
   '</div>',
-  OVERLAY =
+    OVERLAY =
```

Comment:
It is part of a multi-variable `var` statement (which I don't like, but whatever). The linter says that variables in a multi-variable `var` statement that are on not-the-first-line need indented one step. If I had fixed this by hand (instead of with `eslint --fix`, I would have also indented the markup above this. I'll do that now.